pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -534,7 +540,8 @@ func (r *hibernationReconciler) nodesReady(cd *hivev1.ClusterDeployment, remoteC
 	if hibernatingCondition == nil {
 		return false, errors.New("cannot find hibernating condition")
 	}
-	if time.Since(hibernatingCondition.LastProbeTime.Time) < nodeCheckWaitTime {
+	// Don't delay nodeCheckWaitTime if we just discovered SyncSets have been applied
+	if hibernatingCondition.Reason != hivev1.SyncSetsAppliedReason && time.Since(hibernatingCondition.LastProbeTime.Time) < nodeCheckWaitTime {
```

Comment:
another thing that needs to make more sense when the refactor happens - not waiting because syncsets have applied makes no sense :weary:

Reply:
Right... except with the zoomed-out view that SyncSetsNotApplied => SyncSetsApplied is part of the install (as opposed to resume) sequence, where it doesn't make sense to impose the delay designed for resuming.